packages/wonder-blocks-data/src/hooks/use-server-effect.js

```diff
@@ -27,6 +28,10 @@ export const useServerEffect = <TData: ValidCacheData>(
     handler: () => Promise<TData>,
     hydrate: boolean = true,
 ): ?Result<TData> => {
+    // Plug in to the request interception framework for code that wants
+    // to use that.
+    const interceptedHandler = useRequestInterception(requestId, handler);
```

Comment:
The only downside here is that because `useHydratableEffect` implements server-side skip by using an abort and passing it to the handler, it means that can still be intercepted. However, I think that's an unlikely to be surprising scenario - if we find it necessary, we can add a `skip` option to this hook and move the way we handle skips sometime in the future.